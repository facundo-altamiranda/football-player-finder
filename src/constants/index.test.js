import * as constants from './index';

describe('constants', () => {
  it('should match snapshot', () => {
    expect(constants).toMatchSnapshot();
  });
});
