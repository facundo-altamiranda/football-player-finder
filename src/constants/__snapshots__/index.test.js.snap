// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constants should match snapshot 1`] = `
Object {
  "API_URL": "https://football-players-b31f2.firebaseio.com/players.json",
  "POSITION_OPTIONS": Array [
    Object {
      "value": "Attacking Midfield",
    },
    Object {
      "value": "Central Midfield",
    },
    Object {
      "value": "Centre-Back",
    },
    Object {
      "value": "Centre-Forward",
    },
    Object {
      "value": "Defensive Midfield",
    },
    Object {
      "value": "Keeper",
    },
    Object {
      "value": "Left Midfield",
    },
    Object {
      "value": "Left Wing",
    },
    Object {
      "value": "Left-Back",
    },
    Object {
      "value": "Right-Back",
    },
  ],
  "TABLE_COLUMNS": Array [
    Object {
      "dataIndex": "name",
      "key": "name",
      "title": "Player",
    },
    Object {
      "dataIndex": "position",
      "key": "position",
      "title": "Position",
    },
    Object {
      "dataIndex": "nationality",
      "key": "nationality",
      "title": "Nationality",
    },
    Object {
      "dataIndex": "age",
      "key": "age",
      "title": "Age",
    },
  ],
}
`;
